### ISSUE TITLE
How to export the tuned anomaly detection model

### ISSUE THREAD

**Issue by zhangJianfeng** [siddiqmd/FeedbackIsolationForest#2]
I'm very interested in this method and would like to try it in our data. I was wondering is it possible to export a model after several rounds human feedback on the training data and run the exported model on the test data?

**Comment by siddiqmd** [comment]
Yes, its possible but not implemented in the current version. I would suggest that you can write some serialization methods that write the trees into a file/stream and later de-serializes from that file/stream into an Isolation Forest class object.
